includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -1459,11 +1475,280 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
+			|| $this->is_on_add_payment_method_page() ) {
+			return;
+		}
+
+		$order = wc_get_order( $order_id );
+
+		if ( ! $order instanceof WC_Order || ! $order->get_date_paid( 'edit' ) ) {
+			return;
+		}
+
+		$this->record_paid_cart_marker( $order );
+	}
```

PHPMD (2.15.0)

[error] The method handle_paid_order_after_checkout is not named in camelCase. (undefined)

(CamelCaseMethodName)